onnx/defs/schema.h

```diff
@@ -606,6 +606,11 @@ class OpSchema final {
     return numeric_types_for_math_reduction_ir4;
   }
 
+  // Deprecated function, use numeric_types_for_math_reduction_ir4 instread. It will be removed in onnx==1.15.0.
```

Comment:
```suggestion
  // Deprecated function, use numeric_types_for_math_reduction_ir4 instead. It will be removed in onnx==1.15.0.
```
nit: typo. I am surprised that misspell does not catch this somehow